web/static/web/css/base.css

```diff
@@ -9,6 +9,7 @@ html {
 .navbar-item img {
     max-height: 3rem
 }
-:root {
-    --bulma-navbar-height: 4rem !important;
+
+.switch[type=checkbox]:focus+label:before {
```

Comment:
На внешний объект сработало, а на внутренний круг - нет. Я посмотрел, потому что для внутреннего тот же самый стиль, но с `:after`. Заодно я бы сделал не одно двоеточие, а двойное, потому что: 
![image](https://github.com/melomatch/melomatch/assets/90243110/9544d17d-2608-4f03-8492-fc115a848489)

```suggestion
.switch[type=checkbox]:focus+label::after, .switch[type=checkbox]:focus+label::before {
```

Reply:
Добавила, еще предлагаю убрать толщину и сделать `outline: 0 solid #b5b5b5`, как по мне, так выглядит лучше (контур не скачет)

Reply:
Да, давай

Reply:
Стоп, щас понял, что 0 - это просто отсутствие обводки)) Но хоть какая-то она быть должна. Но с дробным числом я не вижу разницу с 1px. Немного почитал про это, видимо браузер просто округляет до целого, соответственно до 1.